kafkabp/consumer.go

```diff
@@ -35,6 +35,13 @@ var (
 	}, rebalanceLabels)
 )
 
+func init() {
+	// Register the error counter so that it can be monitored.
+	rebalanceCounter.With(prometheus.Labels{
+		successLabel: prometheusbp.BoolString(false),
```

Comment:
I think we want to avoid success boolean labels. Prometheus best practices prefers separate "all FOO" and "error FOO" counters. This would avoid the need to do this init here.